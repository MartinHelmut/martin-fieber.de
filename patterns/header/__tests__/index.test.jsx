import React from "react";
import renderer from "react-test-renderer";

import Header from "../index";

describe("Header", () => {
  test("should render", () => {
    const tree = renderer.create(<Header>Top title</Header>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
