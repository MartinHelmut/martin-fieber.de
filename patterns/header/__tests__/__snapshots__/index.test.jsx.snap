// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render 1`] = `
.c0 {
  margin: 0 0 var(--size-600) 0;
}

<header
  className="c0"
>
  Top title
</header>
`;
